remo/reports/models.py

```diff
@@ -246,6 +264,20 @@ def save(self, *args, **kwargs):
         self.user.userprofile.longest_streak_end = longest_end
         self.user.userprofile.save()
 
+    def get_action_items(self):
+        """Returns a list of action items.
+
+        An action item is returned in the case that an activity needs
+        verification from the mentor of a user.
+        """
+
+        if self.activity.is_verifiable and not self.verified_activity:
+            due_date = get_date(weeks=VERIFY_ACTIVITY_WEEKS)
+            name = u'{0} {1}'.format(VERIFY_ACTION, self.user.get_full_name())
+            priority = ActionItem.NORMAL
+            return [Item(name, self.mentor, priority, due_date)]
+        return []
```

Comment:
In order for this to be more readable and uniform according to the rest of ``get_action_items`` method I would prefer some like this:

```
action_items = []
if foo:
    action_items.append(item)
return action_items